nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HelmProxyRecipe.groovy

```diff
@@ -0,0 +1,199 @@
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm.internal.content.recipe
+
+import javax.annotation.Nonnull
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Provider
+import javax.inject.Singleton
+
+import org.sonatype.nexus.repository.Format
+import org.sonatype.nexus.repository.RecipeSupport
+import org.sonatype.nexus.repository.Repository
+import org.sonatype.nexus.repository.Type
+import org.sonatype.nexus.repository.cache.NegativeCacheFacet
+import org.sonatype.nexus.repository.cache.NegativeCacheHandler
+import org.sonatype.nexus.repository.http.PartialFetchHandler
+import org.sonatype.nexus.repository.httpclient.HttpClientFacet
+import org.sonatype.nexus.repository.proxy.ProxyHandler
+import org.sonatype.nexus.repository.routing.RoutingRuleHandler
+import org.sonatype.nexus.repository.security.SecurityHandler
+import org.sonatype.nexus.repository.types.ProxyType
+import org.sonatype.nexus.repository.view.ConfigurableViewFacet
+import org.sonatype.nexus.repository.view.Context
+import org.sonatype.nexus.repository.view.Matcher
+import org.sonatype.nexus.repository.view.Route
+import org.sonatype.nexus.repository.view.Router
+import org.sonatype.nexus.repository.view.ViewFacet
+import org.sonatype.nexus.repository.view.handlers.ConditionalRequestHandler
+import org.sonatype.nexus.repository.view.handlers.ContentHeadersHandler
+import org.sonatype.nexus.repository.view.handlers.ExceptionHandler
+import org.sonatype.nexus.repository.view.handlers.HandlerContributor
+import org.sonatype.nexus.repository.view.handlers.TimingHandler
+import org.sonatype.nexus.repository.view.matchers.ActionMatcher
+import org.sonatype.nexus.repository.view.matchers.LiteralMatcher
+import org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher
+import org.sonatype.repository.helm.internal.AssetKind
+import org.sonatype.repository.helm.internal.HelmFormat
+import org.sonatype.repository.helm.internal.content.HelmContentFacet
+import org.sonatype.repository.helm.internal.security.HelmSecurityFacet
+
+import static org.sonatype.nexus.repository.http.HttpHandlers.notFound
+import static org.sonatype.nexus.repository.http.HttpMethods.GET
+import static org.sonatype.nexus.repository.http.HttpMethods.HEAD
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_INDEX
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PACKAGE
+
+/**
+ * @since 1.0.10
+ */
+@Named(HelmProxyRecipe.NAME)
+@Singleton
+class HelmProxyRecipe
+    extends RecipeSupport
+{
+  public static final String NAME = 'helm-proxy'
+  public static final String PATH_NAME = 'path'
```

Comment:
Not used variable

Reply:
Thanks, fixed 5aad649f943e777daa53bf0b4b845e75636af662